.github/workflows/on-release.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow is triggered when someone create a new release in GitHub
+# and checks the "This is a pre-release" box.
+name: Trigger Release creation
+
+on:
+  release:
+    types: [prereleased]
+
+jobs:
+  on-release:
+    runs-on: ubuntu-latest
+    name: Release module
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set environment variables from parameters
+        shell: bash
+        run: |
+          echo "NEXUS_USERNAME=${{ secrets.NEXUS_USERNAME }}" >> $GITHUB_ENV
+          echo "NEXUS_PASSWORD=${{ secrets.NEXUS_PASSWORD }}" >> $GITHUB_ENV
+      # Setting up the SSH agent to be able to commit back to the repository
+      # https://github.com/webfactory/ssh-agent
+      - uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.GH_SSH_PRIVATE_KEY_JAHIACI }}
+      - name: Delete github tag ${{ github.event.release.tag_name }}
+        shell: bash
+        run: |
+          git config user.email 'jahia-ci@jahia.com'
+          git config user.name 'Jahia CI'
+          git fetch --all
+          git tag --delete ${{ github.event.release.tag_name }}
+          git push origin :refs/tags/${{ github.event.release.tag_name }}
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: 'temurin'
+      - name: Release with Maven
+        run: |
+          TAG_VERSION=$(echo ${{ github.event.release.tag_name }}) # e.g. 2.20.12-jahia1
+          FINAL_RELEASE_VERSION=$(echo ${{ tag_name }} | tr '_' '.')
+          REVISION=$(echo $FINAL_RELEASE_VERSION | tr -d 'jahia' | cut -d'-' -f2)
```

Comment:
I think this will not work with the format of Cellar version: cellar-4.1.3-jahia10